tests/test_tokenization_wav2vec2.py

```diff
@@ -470,3 +495,55 @@ def test_special_characters_in_vocab(self):
     def test_pretrained_model_lists(self):
         # Wav2Vec2Model has no max model length => no testing
         pass
+
+    # overwrite from test_tokenization_common
+    def test_add_tokens_tokenizer(self):
```

Comment:
Wav2Vec2 should split on newly added tokens that are longer than 1 char:
Doing:

`tok.add_tokens("aaa")`

Should then encode / decode to

`tok.decode(tok("hello this is to encode aaa").input_ids) == "hello this is to encode aaa"`

and not 

`tok.decode(tok("hello this is to encode aaa").input_ids) == "hello this is to encodeaaa"`